easybuild/easyconfigs/l/lightning/lightning-2.2.0.post0-foss-2023a-PyTorch-2.1.2-CUDA-12.1.1.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'PythonPackage'
+
+name = 'lightning'
```

Comment:
@NPotvin Isn't this basically the same thing as `PyTorch-Lightning-2.2.0.post0-foss-2023a-CUDA-12.1.1.eb`?

Or is there actually something additional installed here on top of `PyTorch-Lightning`?

Reply:
@boegel It's an additional layer on top of PyTorch lightning. It is developed by the same team, apparently. On pypi lightning and pytorch-lightning are two different packages.
(have a look [here](https://github.com/Lightning-AI/pytorch-lightning))

Reply:
Zero room for confusion there... 🙈